RELEASE.md

```diff
@@ -66,6 +66,27 @@ Please make sure you have these tools installed before starting this process:
 
     Your chart will be deployed to [the release environment described in this table](./README.md)
 
+-   [ ] 7.1.  Cherry-pick fixes
+
+    At this stage, it is possible for the build to have failed. The fixes for this failure should have been merged to `master` following a successfully approved Pull Request.
+
+    You can cherry-pick the required changes with:
+
+    ```bash
+    $ git cherry-pick <commit-hash>
+    ```
+
+    Ensure the cherry-pick commit is free from any conflicts.
+
+-   [ ] 7.2.  Push new Release Candidate tag
+
+    Since there are new hotfix changes to the release, you must then add another release candidate tag to your release branch and push it again.
+
+    ```bash
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-shell-dollars] Do not use dollar signs before shell-commands](https://app.codacy.com/gh/codacy/chart/pullRequest?prid=5562888)